tools/photodiode_experiment.py

```diff
@@ -0,0 +1,258 @@
+from psychopy import visual, core
+import serial
+from pylsl import StreamInfo, StreamOutlet
+
+"""PsychoPy Photodiode Experiment for Clock Synchronization.
+
+This script displays a simple visual stimulus (e.g., a flashing square)
+intended to be captured by a photodiode. Its primary purpose is to generate
```

Comment:
It may be cleaner to separate the experiment from the recording. Could you create a main.py? Then, it can handle recording and/or presenting. This file will only handle the triggers and presentation.

It might be good to turn off one or the other triggers. If a user has no MMTB, for instance, how could they still run or configure using this?